rpcserver.go

```diff
@@ -2833,9 +2836,11 @@ func (r *rpcServer) CloseChannel(in *lnrpc.CloseChannelRequest,
 		// If the user hasn't specified NoWait, then before we attempt
 		// to close the channel we ensure there are no active HTLCs on
 		// the link.
-		if !in.NoWait && len(channel.ActiveHtlcs()) != 0 {
-			return fmt.Errorf("cannot co-op close channel " +
-				"with active htlcs")
+		if !in.NoWait && len(activeHtlcs) != 0 {
+			return fmt.Errorf("cannot co-op close channel with "+
+				"active htlcs (number of active htlcs: %d), "+
+				"coop close nonetheless by setting "+
```

Comment:
not sure if that's the correct usage of `nonetheless`...I think you mean `bypass this check by setting no_wait=true`?